examples/with-nx/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "scripts": {
+    "start": "nx serve",
+    "build": "nx build",
+    "test": "nx test"
+  },
+  "private": true,
+  "dependencies": {
+    "@nrwl/next": "14.6.3",
+    "core-js": "^3.6.5",
+    "next": "12.2.5",
+    "react": "18.2.0",
+    "react-dom": "18.2.0",
+    "regenerator-runtime": "0.13.7",
+    "tslib": "^2.3.0"
```

Comment:
Are `core-js`, `regenerator-runtime`, and `tslib` needed as dependencies?

Reply:
Good catch, this was generated from a boilerplate `npx create-nx-workspace` so some of this looks to be unused for the simpler cases.